connection_go18.go

```diff
@@ -162,7 +162,7 @@ func (mc *mysqlConn) watchCancel(ctx context.Context) error {
 	case <-ctx.Done():
 		killErr := mc.kill()
 		if killErr != nil {
-			errLog.Print("failed to kill query: " + killErr.Error())
+			errLog.Printf("failed to kill query: %s", killErr)
 		}
```

Comment:
sorry, irrelevant  comment in previous path.
could you please make this into:
`errLog.Print("failed to kill query: ", killErr)`

Reply:
ok